cms/djangoapps/contentstore/views/course.py

```diff
@@ -608,6 +608,21 @@ def _deprecated_blocks_info(course_module, deprecated_block_types):
     return data
 
 
+def _get_proctored_exam_settings_url(course_module):
```

Comment:
I think if we move this function into `contentstore/utils.py`, then this function can be used in this file and in `contentstore/views/certificates.py` as well.